jib-core/src/main/java/com/google/cloud/tools/jib/cache/Cache.java

```diff
@@ -66,32 +69,52 @@ private Cache(CacheStorageFiles cacheStorageFiles) {
   }
 
   /**
-   * Saves a manifest and container configuration for a V2.2 or OCI image.
+   * Saves image metadata (a manifest list and a list of manifest/container configuration pairs) for
+   * an image reference.
    *
-   * @param imageReference the image reference to save the manifest and container configuration for
-   * @param manifestTemplate the V2.2 or OCI manifest
-   * @param containerConfigurationTemplate the container configuration
+   * @param imageReference the image reference to save the metadata for
+   * @param metadata the image metadata
+   * @throws IOException if an I/O exception occurs
+   */
+  public void writeMetadata(ImageReference imageReference, ImageMetadataTemplate metadata)
+      throws IOException {
+    cacheStorageWriter.writeMetadata(imageReference, metadata);
+  }
+
+  /**
+   * Saves a schema 2 manifest for an image reference. This is a simple wrapper around {@link
+   * #writeMetadata(ImageReference, ImageMetadataTemplate)} to save a single manifest without a
+   * manifest list.
+   *
+   * @param imageReference the image reference to save the manifest for
+   * @param manifest the V2.2 or OCI manifest
```

Comment:
I thought we also write 2.1 manifests down to cache?

Reply:
Yes, we have a dedicated API for V2.1 on  line 113 (line 92 in old code).